apps/src/aichat/types.ts

```diff
@@ -59,6 +59,11 @@ export enum ViewMode {
   PRESENTATION = 'presentation-mode',
 }
 
+export enum WorkspaceTeacherViewTab {
```

Comment:
Minor - this could just stay in `ChatWorkspace.tsx` since it's only used internally by that component